.github/workflows/ci.yml

```diff
@@ -103,7 +103,7 @@ jobs:
           docker pull ghcr.io/ansys/ansys-api-turbogrid/tglin_reduced:241    
 
       - name: Create Key
-        uses: mobiledevops/secret-to-file-action@v1
+        uses: Acohen-work/secret-to-file-action@v1
```

Comment:
Alex, we can not have personal things in the project

Reply:
No license in the repo

Reply:
No license does not mean you can distribute. If you need this action, it should be moved to the ansys org and go under the OSS process. Since this is very specific of this project, the best way to speed up things is that you emulate the goal of this action in a shell script that is hosted in this repository.

Reply:
This suggestion sounds reasonable to me.

Reply:
@jorgepiloto the action is a node.js script which handles the secrecy details of the file as well. If you have another route to securely handling secret->file then we can go that way, if not, why not simply fork this action into ansys actions? It surely is generically useful.

Reply:
If you want to achieve the same result, why not doing the following:

```shell
base64 -d <<< ${{ secrets.CONTAINER_ACCESS_KEY }} > tg_container_key
```